app/src/main/java/org/sopt/havit/ui/home/HomeFragment.kt

```diff
@@ -91,10 +91,10 @@ class HomeFragment : BaseBindingFragment<FragmentHomeBinding>(R.layout.fragment_
     }
 
     private fun setData() {
-        homeViewModel.requestUserDataTaken()
-        homeViewModel.requestContentsTaken()
-        homeViewModel.requestCategoryTaken()
-        homeViewModel.requestRecommendTaken()
+        homeViewModel.requestUserDataTaken()    // 도달률
+        homeViewModel.requestContentsTaken()    // 최근 저장 콘텐츠
+        homeViewModel.requestCategoryTaken()    // 카테고리
+        homeViewModel.requestRecommendTaken()   // 추천 콘텐츠
```

Comment:
homeViewModel 코드 중복이 있는데 이런식으로 바꿔보는 건 어떨까요? (안하셔두 됨니닷)
```suggestion
with(homeViewModel) {
        requestUserDataTaken()    // 도달률
        requestContentsTaken()    // 최근 저장 콘텐츠
        requestCategoryTaken()    // 카테고리
        requestRecommendTaken()   // 추천 콘텐츠
}

Reply:
반영하겠습니닷!-!